visualization/app/codeCharta/ui/codeMap/rendering/codeMapBuilding.ts

```diff
@@ -70,7 +72,10 @@ export class CodeMapBuilding {
 
 	resetColor() {
 		this._color = this._defaultColor
-		this._deltaColor = this._defaultDeltaColor
+		// this._deltaColor = this._defaultDeltaColor
```

Comment:
```suggestion
```

Reply:
> 

Sorry for not noticing this suggested change